ansible/templates/dehydrated.j2

```diff
@@ -0,0 +1,17 @@
+##########################################################################
+# See the dehydrated docs for all config options:
+# https://github.com/lukas2511/dehydrated/blob/master/docs/examples/config
+##########################################################################
+
+# Path to certificate authority (default: https://acme-v01.api.letsencrypt.org/directory)
+#CA="https://acme-v01.api.letsencrypt.org/directory"
+
+# Path to certificate authority license terms redirect (default: https://acme-v01.api.letsencrypt.org/terms)
+#CA_TERMS="https://acme-v01.api.letsencrypt.org/terms"
+
+# Staging servers, for testing
+CA="https://acme-staging.api.letsencrypt.org/directory"
```

Comment:
I like having the commented-out staging vars, in case someone needs to make a change and test it. It'll be too easy for that person to not even know there are different values for testing.